### DISCUSSION TITLE
anonymous read-only access and LDAP auth

### DISCUSSION THREAD

**Discussion by Kludge-As-A-Service** [netbox-community/netbox#8012] [Q&A] [answered]
Hello, we're using Netbox 3.0.7 with LDAP auth, it works fine except that anonymous users can only access the main dashboard and nothing else (padlocks showing on every section in the left pane, and 403 error if we try to access them directly from a URL). I would like anonymous users to have read-only access, as is the intended default behavior I believe. I have LOGIN_REQUIRED = False in my configuration.py file. Any idea?
Thanks.

**Comment by jeremystretch** [comment, marked as answer]
You can configure the set of desired views by defining the [`EXEMPT_VIEW_PERMISSIONS`](https://netbox.readthedocs.io/en/stable/configuration/optional-settings/#exempt_view_permissions) config parameter in `configuration.py`. To allow anonymous access to all views, do:

```
EXEMPT_VIEW_PERMISSIONS = ['*']
```

**Reply by Kludge-As-A-Service** [reply]
It works, thanks!
